src/util/triplea/mapXmlCreator/MapXmlData.java

```diff
@@ -0,0 +1,365 @@
+package util.triplea.mapXmlCreator;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
+import games.strategy.util.Triple;
+import util.triplea.mapXmlCreator.TerritoryDefinitionDialog.DEFINITION;
+
+public class MapXmlData {
+  private String notes = "";
+  private File mapXMLFile;
+
+  private Map<String, String> xmlStringsMap = Maps.newLinkedHashMap();
+  private List<String> resourceList = new ArrayList<String>();
+  private Map<String, HashMap<DEFINITION, Boolean>> territoryDefintionsMap =
+      new HashMap<String, HashMap<DEFINITION, Boolean>>();
+  private Map<String, Set<String>> territoryConnectionsMap = new HashMap<String, Set<String>>();
+  private List<String> playerNames = new ArrayList<String>();
+  private Map<String, String> playerAllianceMap = new HashMap<String, String>();
+  private Map<String, Integer> playerInitResourcesMap = new HashMap<String, Integer>();
+  private Map<String, List<Integer>> unitDefinitionsMap = Maps.newLinkedHashMap();
+  private Map<String, List<String>> gamePlaySequenceMap = Maps.newLinkedHashMap();
+  private Map<String, Triple<String, String, Integer>> playerSequenceMap =
+      Maps.newLinkedHashMap();
+  private Map<String, List<String>> technologyDefinitionsMap =
+      Maps.newLinkedHashMap();
+  private Map<String, List<String>> productionFrontiersMap = Maps.newLinkedHashMap();
+  private Map<String, List<String>> unitAttatchmentsMap = Maps.newLinkedHashMap();
+  private Map<String, Integer> territoyProductionsMap = new HashMap<String, Integer>();
+  private Map<String, CanalTerritoriesTuple> canalDefinitionsMap =
+      new HashMap<String, CanalTerritoriesTuple>();
+  private Map<String, String> territoryOwnershipsMap = new HashMap<String, String>();
+  private Map<String, Map<String, Map<String, Integer>>> unitPlacementsMap =
+      new HashMap<String, Map<String, Map<String, Integer>>>();
+  private Map<String, List<String>> gameSettingsMap = new HashMap<String, List<String>>();
+
+  public MapXmlData() {
+    initialize();
+  }
+
+  @SuppressWarnings("unused")
```

Comment:
Why the suppress warnings?
Test cannot access 'private' functions, so either this should be used or can be axed.